content/docs/sidebar.json

```diff
@@ -196,6 +196,53 @@
         "label": "doctor",
         "slug": "doctor"
       },
+      {
+        "label": "experiments",
+        "slug": "exp",
+        "source": "exp/index.md",
```

Comment:
Not sure about the abbreviated slug and directory. The directory doesn't matter that much (it may just be easier if it matches the slug) but the slug becomes part of the URL which may impact SEO a bit. Idk

Reply:
On the other hand I like the shorter version and having both in the left nav makes it more obvious that you can use either form:

![image](https://user-images.githubusercontent.com/1477535/106070065-5d9a0500-60c9-11eb-82a1-b30b938143a7.png)

WDYT @shcheklein

Reply:
Cc @dberenbaum

Reply:
Makes sense to me!